### ISSUE TITLE
Tor configuration using `streamisolation`

### ISSUE THREAD

**Issue by bjarnemagnussen** [Stadicus/RaspiBolt#487]
Hi,

I just wanted to mention that in the official LND documentation regarding Tor (https://github.com/lightningnetwork/lnd/blob/master/docs/configuring_tor.md) they advise to activate `streamisolation` like this:

```
[Tor]
tor.active=1
**tor.streamisolation=true**
tor.socks=localhost
tor.v3=true
```

Is there any specific reason for why inside the Raspibolt Tor documentation (`raspibolt_69_tor.md`) the option `streamisolation` is ignored?

**Comment by Septem151** [comment]
According to the commit dates that `streamisolation` was added to the lnd.conf documentation - https://github.com/lightningnetwork/lnd/commit/4cba9dc8d3bae93f4e3478c46dca8b41cfb1b619#diff-2dad73b2f802ce6806e12e8d5dbeeaaf - this addition predates the creation of the RaspiBolt guide. 

@Stadicus should Tor users be setting `tor.streamisolation=true` in the lnd.conf file when running RaspiBolt, or is there an optimization issue with rPi's that prevents setting to true?

**Comment by kristapsk** [comment]
Didn't notice this issue before, but I have addressed this in #496. Haven't noticed any performance issues with this with RPi 3 so far.

**Comment by Stadicus** [comment]
Thanks, just merged it.
